src/source.py

```diff
@@ -16,4 +16,82 @@ def d2YdtdE_0_func_doublegauss(t, E, N, mu_t, sig_t, mu_E, sig_E):
 
 def dYdt_0_func_gauss(t, E, N, mu_t, sig_t):
     dYdt_0 = N/np.sqrt(2*np.pi)/sig_t * np.exp(-0.5 * ((t-mu_t)/sig_t)**2)
-    return dYdt_0
\ No newline at end of file
+    return dYdt_0
+
+# -*- coding: utf-8 -*-
+"""
+Created on Mon Feb 20 09:27:10 2023
+
+@author: blr
+"""
+import numpy as np
+import scipy
```

Comment:
It looks like something went weird here with the header where it got repeated, along with my two functions? Unless I'm misunderstanding how to read pull requests lol